script/utils/start-backing-services

```diff
@@ -17,5 +17,11 @@ if [ "$MOCK_API" = true ]; then
   npx ts-node --transpile-only ./wiremock/scripts/resetStubs.ts > /dev/null
   npx ts-node --transpile-only ./wiremock/scripts/stubApis.ts > /dev/null
 else
-  docker compose up --scale=wiremock=0 -d
+  docker compose up -d
+
+  echo "==> Stubbing Prisoner Offender Search endpoints..."
+
+  npx ts-node --transpile-only ./wiremock/scripts/resetStubs.ts > /dev/null
```

Comment:
Ha, just after I de-abstracted it! We could make this a method in this file, but it's not a big deal right now